internal/gps/strip_vendor.go

```diff
@@ -12,10 +12,6 @@ import (
 )
 
 func stripVendor(path string, info os.FileInfo, err error) error {
-	if err != nil && err != filepath.SkipDir {
```

Comment:
Because of the path passed causes `os.Lstat` to return an error somehow, then `walkFn` will be passed `nil` for `info` and the error. Jumping to the code below will cause the panic again.